src/components/CryptoCards/CryptoCards.styles.tsx

```diff
@@ -0,0 +1,177 @@
+import resetCSS from '../../styles/reset';
+import fonts from '../../styles/fonts';
+import color from '../../styles/colors';
+import { css } from 'styled-components';
+import { getShade } from '../../styles/colors';
+
+const borderStyle = css`
+    ${resetCSS}
+    height: 336px;
+    width: 265px;
+    left: 20px;
+    top: 20px;
+    border-radius: 16px;
+    padding: 24px, 40px, 24px, 40px;
+    border: 1px solid ${getShade('light', 10)};
+    box-sizing: border-box;
+    border-radius: 16px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 24px 40px;
+    position: absolute;
+    width: 265px;
+    height: 336px;
+    left: 20px;
+    top: 20px;
```

Comment:
Redudant css code

Reply:
I'm not sure why this is redundant, let me check the MR fully, but i'm sure Loco is correct. One thing i would say is we are using a VSCode plugin called 'sort lines' we use this to alphabetise props in CSS and JS, if you could do that to align, i would be vert grateful. TY

Reply:
Look at padding for example its there twice

Reply:
done